mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -404,13 +457,15 @@ def compute_next_token(current_size):
             offset = SearchUtils.parse_start_offset_from_page_token(page_token)
             lifecycle_stags = set(LifecycleStage.view_type_to_stages(view_type))
 
+            experiment_filters = [
+                *attribute_filters,
+                SqlExperiment.lifecycle_stage.in_(lifecycle_stags),
```

Comment:
`lifecycle_stages`? Can we just fix this typo since we're poking around in these variables anyway?